### PR TITLE
[Paddle]Add chapter_computer-vision

### PR DESCRIPTION
**Pull request by tngt**

### CODE REVIEW

File: chapter_computer-vision/semantic-segmentation-and-dataset.md [hunk 1]
```diff
@@ -231,6 +302,16 @@ imgs = [img.permute(1, 2, 0) for img in imgs]
 d2l.show_images(imgs[::2] + imgs[1::2], 2, n);
 ```
 
+```{.python .input}
+#@tab paddle
+imgs = []
+for _ in range(n):
+    imgs += voc_rand_crop(train_features[0], train_labels[0], 200, 300)
+
+imgs = [img.transpose([1,2,0]).astype(paddle.uint8) for img in imgs]
```
Comment (cheungdaven) [inline chapter_computer-vision/semantic-segmentation-and-dataset.md:311]: [1, 2, 0]

**Review by cheungdaven — CHANGES_REQUESTED** [review]

**Comment by cheungdaven** [comment]
Could you also tune the performance of paddlepaddle in ssd.md.

File: chapter_computer-vision/anchor.md [hunk 2]
```diff
@@ -474,6 +596,41 @@ def multibox_target(anchors, labels):
...
+def multibox_target(anchors, labels):
+    """使用真实边界框标记锚框"""
+    batch_size, anchors = labels.shape[0], anchors.squeeze(0)
+    batch_offset, batch_mask, batch_class_labels = [], [], []
+    place, num_anchors = anchors.place, anchors.shape[0]
+    for i in range(batch_size):
+        label = labels[i, :, :]
+        anchors_bbox_map = assign_anchor_to_bbox(
+            label[:, 1:], anchors, place)
+        bbox_mask = paddle.tile(paddle.to_tensor((anchors_bbox_map >= 0), dtype='float32').unsqueeze(-1),(1,4))
```
[the remark this answers is not in the document]
Comment (w5688414) [reply chapter_computer-vision/anchor.md:611]: Done

File: chapter_computer-vision/fcn.md [hunk 3]
```diff
@@ -248,6 +326,16 @@ trainer = torch.optim.SGD(net.parameters(), lr=lr, weight_decay=wd)
 d2l.train_ch13(net, train_iter, test_iter, loss, trainer, num_epochs, devices)
 ```
 
+```{.python .input}
+#@tab paddle
+def loss(inputs, targets):
+    return F.cross_entropy(inputs.transpose([0,2,3,1]), targets, reduction='none').mean(1).mean(1)
```
[the remark this answers is not in the document]
Comment (w5688414) [reply chapter_computer-vision/fcn.md:332]: Done

File: chapter_computer-vision/fcn.md [hunk 4]
```diff
@@ -321,6 +425,21 @@ for i in range(n):
...
+voc_dir = d2l.download_extract('voc2012', 'VOCdevkit/VOC2012')
+test_images, test_labels = d2l.read_voc_images(voc_dir, False)
+n, imgs = 4, []
+for i in range(n):
+    crop_rect = (0, 0, 320, 480)
+    X = paddlevision.transforms.crop(test_images[i], *crop_rect)
+    pred = label2image(predict(X))
+    imgs += [X.transpose([1,2,0]).astype('uint8'), pred,
+             paddlevision.transforms.crop(
+                 test_labels[i], *crop_rect).transpose([1,2,0]).astype("uint8")]
```
[the remark this answers is not in the document]
Comment (w5688414) [reply chapter_computer-vision/fcn.md:439]: Done

File: chapter_computer-vision/semantic-segmentation-and-dataset.md [hunk 5]
```diff
@@ -116,6 +148,14 @@ imgs = [img.permute(1,2,0) for img in imgs]
 d2l.show_images(imgs, 2, n);
 ```
 
+```{.python .input}
+#@tab paddle
+n = 5
+imgs = train_features[0:n] + train_labels[0:n]
+imgs = [img.transpose([1,2,0]).astype(paddle.uint8) for img in imgs]
```
[the remark this answers is not in the document]
Comment (w5688414) [reply chapter_computer-vision/semantic-segmentation-and-dataset.md:155]: Done

**Comment by w5688414** [comment]
image
 
I fixed the issue and solved the conflicts, re run the ssd section, the result seems ok right now, can you review this chapter again? @cheungdaven thanks

**Pull request merged**